src/helm/proxy/clients/huggingface_model_registry.py

```diff
@@ -10,6 +10,8 @@
     FULL_FUNCTIONALITY_TEXT_MODEL_TAG,
 )
 
+# The path where local HuggingFace models should be downloaded or symlinked, e.g. ./helm-models/llama-7b
+LOCAL_MODEL_DIR = "./helm-models"
```

Comment:
`"./huggingface_models"`?

Reasoning:

- `_` for consistency with `prod_env` and `benchmark_output`
- `huggingface` for clarity that this only supports huggingface format
- Don't need `helm` because we are already in a HELM working directory